ideas/146-status-go-sdk/README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Could you add yaml front matter for this one? Don't need to edit README anymore (will remove soon)